### DISCUSSION TITLE
Export only photos from immich

### DISCUSSION THREAD

**Discussion by radityakurnianto** [immich-app/immich#9660] [General]
I just wondering,  is it ok to just copy all images from `/upload` folder from immich directory?

**Comment by bo0tzz** [comment]
Yes

**Reply by radityakurnianto** [reply]
Next question, when I `cd` to the user id of upload path and execute this command `tree -a` it return different count from select all menu in immich dashboard.

As a comparison please check this screenshot

files count using select all menu  -> `return 10280`
![image](https://github.com/immich-app/immich/assets/35289964/c5c15b08-94cc-421f-811f-94ce824d83fd)


files count using `tree -a` -> `return 10351`
![image](https://github.com/immich-app/immich/assets/35289964/c6f66519-ae64-45c4-bd66-be10467fae9d)


if using `find . -name '*.*' | wc -l` -> `return 10352` 

`-name '*.*'` is used since the upload path stored various extension such HEIC, MOV, jpg, etc
![image](https://github.com/immich-app/immich/assets/35289964/d5dab1fd-d2b9-4c2a-aa58-0d61386ddfea)

are there anything wrong with my method to count?

what I want to achieve is, to simply copy every file inside `/upload` path to `my_path` without using the download feature from immich.
